934fe8c/README.md

```diff
@@ -283,6 +283,153 @@ The trusted-setup coordinator will give each participant an `upload_url` which c
 $ rsync -vP --append -e 'ssh -i <path to SSH private-key>' <files> <upload url>
 ```
 
+## Phase 2 Validation
+
+**Note**: The verification process is very computationally intensive. For details see in the [Hardware Requirements](#hardware-requirements) section the "Contribution Verification" requirements.
+
+1. Build `phase2`. Please follow the instruction from the [Install Dependencies and Build `phase2`](#03-install-dependencies-and-build-phase2) section and stop right before the "Build `phase2`" step. Instead do the following:
+
+```console
+$ git clone https://github.com/filecoin-project/filecoin-phase2.git
+$ cd filecoin-phase2
+$ git checkout 934fe8c6d2df2589644302579838976d070c48a7
+$ cargo build --release --bins
+$ mv target/release/filecoin-phase2 phase2
+```
+
+2. Download verification scripts:
+
+```console
+TODO vmx 2022-01-20
```

Comment:
Maybe something like?
```
$ for f in verify_all.sh download_prereqs_for_initial_generation.sh generate_initial.sh download_prereqs_for_contrib.sh download_prereqs_for_final.sh verify_contrib.sh verify_final.sh; do \
      curl -O "https://raw.githubusercontent.com/filecoin-project/phase2-attestations/master/934fe8c/${f}" && chmod +x "${f}"; \
 done
```

Reply:
Yes, but I'd like to pin it to a commit. Hence I'll add it, once this is merged.